src/parser.ts

```diff
@@ -299,15 +301,22 @@ export class Parser {
         rootExp = typeSymbol;
         const expName = rootExp.getName();
 
-        if (
-          expName === '__function' ||
-          expName === 'StatelessComponent' ||
-          expName === 'Stateless' ||
-          expName === 'StyledComponentClass' ||
-          expName === 'StyledComponent' ||
-          expName === 'FunctionComponent' ||
-          expName === 'ForwardRefExoticComponent'
-        ) {
+        const defaultComponentTypes = [
```

Comment:
Maybe put this list globally in module?
To avoid duplicationg

Reply:
I have thought about that, but list of items is not the same. Since I did dive deep to the implementation I decided to keep them as it is, just refactor to be able easily to extend (array way).

Reply:
Only difference I see is 'default' which can be added in computeComponentName
Anyway, I like solution 👍